src/main/resources/anniv6Resources/localization/eng/UIstrings.json

```diff
@@ -20,6 +20,12 @@
       "Heal "
     ]
   },
+  "${ModID}:AnyColorCardReward": {
+    "TEXT": [
+      "Add a card of any color to deck",
```

Comment:
```suggestion
      "Add a card of any color to your deck",
```

Reply:
Although the current version looks like an oversight, it was actually a conscious decision on my part due to space restraints - see the suggested version in-game, the k doesn't quite fit in the reward box:
![image](https://github.com/erasels/SpireMapOverhaul/assets/85409980/f095770c-16ef-4e1a-9b74-ffbc9d336564)